File: Front/Provent-App/src/app/models/Speaker.ts
import { Event } from "./Event";
import { SocialNetwork } from "./SocialNetwork";

export interface Speaker {
    id: number;
    name: string;
    miniResume: string;
    imageURL: string;
    phone: string;
    email: string;
    socialNetworks: SocialNetwork[];
    speakersEvents: Event[];
}

File: Back/src/Provent.Domain/Speaker.cs
using System.Collections.Generic;

namespace Provent.Domain
{
    public class Speaker
    {
        public int Id { get; set; }
        public string Name { get; set; }
        public string MiniResume { get; set; }
        public string ImageURL { get; set; }
        public string Phone { get; set; }
        public string Email { get; set; }
        public IEnumerable<SocialNetwork> SocialNetworks { get; set; }
        public IEnumerable<SpeakerEvent> SpeakersEvents { get; set; }
    }
}